fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -1408,8 +1426,10 @@ private void releaseTabletCtx(TabletSchedCtx tabletCtx, TabletSchedCtx.State sta
     private synchronized void removeTabletCtx(TabletSchedCtx tabletCtx, String reason) {
         runningTablets.remove(tabletCtx.getTabletId());
         allTabletIds.remove(tabletCtx.getTabletId());
-        schedHistory.add(tabletCtx);
-        LOG.info("remove the tablet {}. because: {}", tabletCtx.getTabletId(), reason);
+        if (reason != null) {
+            schedHistory.add(tabletCtx);
```

Comment:
Should `schedHistory.add(tabletCtx);` out of the if clause?

Reply:
this logic is not used , i will resume